file_paths.md

```diff
@@ -222,13 +222,12 @@ create a `puppet` user or group.
         etc *
             pxp-agent.conf                    # pxp-agent configuration file
             modules *                         # stores configuration files for pxp-agent modules
-                pxp-module-puppet.conf        # configuration file of the pxp module puppet
+                pxp-module-puppet.conf        # configuration file of the pxp module puppet (if required)
```

Comment:
Can probably change `(if required)` to `(optional)` or `(optionally, to override puppet.bat location)`